packages/accordions/src/elements/stepper/components/Label.tsx

```diff
@@ -12,10 +12,15 @@ import { StyledLabel, StyledLabelText, StyledIcon, StyledIconFlexContainer } fro
 import { useStepContext, useStepperContext } from '../../../utils';
 
 interface ILabel {
+  /** Used as the visual object for the stepper */
   icon?: React.ReactNode;
+  /** Determines if the label is displayed */
   isHidden?: boolean;
 }
 
+/**
+ * @extends HTMLDivElement
```

Comment:
Should be `HTMLAttributes<HTMLDivElement>`

Reply:
Fixed.

Reply:
extends have been removed for now.